src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ModuleInitializerAttribute.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Used to indicate to the compiler that a method should be called
+    /// in its containing module's initializer.
+    /// </summary>
+    /// <remarks>
+    /// When one or more valid methods
+    /// with this attribute are found in a compilation, the compiler will
+    /// emit a `&lt;Module&gt;..cctor` which calls each of the attributed methods.
```

Comment:
```suggestion
    /// emit a module initializer which calls each of the attributed methods.
```

`<Module>..cctor` is implementation detail of the compiler.